server/prometheus.js

```diff
@@ -33,15 +44,19 @@ class Prometheus {
         }
     }
 
-    update(heartbeat) {
+    update(heartbeat, tlsInfo) {
+
+        // TODO: TLS Info here
+        console.log(tlsInfo)
+
```

Comment:
I passed the tls info object here, you can get info from this object

Reply:
Thanks, I'll take a look at this a bit later on today